app/helpers/admin/orders_helper.rb

```diff
@@ -0,0 +1,14 @@
+module Admin
+  module OrdersHelper
+    def order_adjustments(order)
```

Comment:
this is not really order_adjustments right?
this method is order_adjustments_displayed_in_the_adjustments_table_on_the_order_edit_page_which_excludes_shipping_method_adjustments_and_zero_tax_rate_adjustments 🤣 

because of this I'd call it order_adjustments_for_display and add a method comment with something like
"we exclude shipping method adjustments because they are displayed in a separate table together with the order line items"
and "we exclude tax rate adjustment with zero value because..." I actually dont know why tax rates need to be excluded here, they would be removed already as not eligible if amount is zero.

Reply:
Thank you for bringing it up. I guess you could argue that helpers are to be used in views. So all helper are "for display" and therefore the suffix is redundant. :stuck_out_tongue_closed_eyes: 
In this case it's probably better to point it out though. 

I had a similar thought about the zero value adjustments. So I first covered that part with a spec and then removed the redundant logic. Much simpler, isn't it?

Reply:
you refactored code to delete it afterwards :-)
re removing tax rate filter, we can do that. I am just not 100% sure it is redundant...